src/toil/test/provisioners/gceProvisionerTest.py

```diff
@@ -287,7 +286,7 @@ def _runScript(self, toilOptions):
         # instances
         runCommand = ['/home/venv/bin/python', '/home/sort.py', '--fileToSort=/home/s3am/bin/asadmin', '--sortMemory=0.6G', '--mergeMemory=3.0G']
         runCommand.extend(toilOptions)
-        runCommand.append('--sseKey=/home/keyFile')
+        #runCommand.append('--sseKey=/home/keyFile')
```

Comment:
I guess google job store doesn't work with sse? Is there an issue for this?

Reply:
SSE keys are not yet supported in Google. Partially because they are not tested. I pushed adding it to the refactor (see #2095).